Misc/NEWS.d/next/Core and Builtins/2024-01-07-04-22-51.gh-issue-108362.oB9Gcf.rst

```diff
@@ -0,0 +1,13 @@
+Implements an incremental cyclic garbage collector. By collecting the old
```

Comment:
Our "house style" for these changelog comments would use "Implement" (and "Reduce" in the next paragraph) instead of "Implements" and "Reduces". (Sorry, I don't know what the grammar experts call the two different tenses or moods or whatevers. :-)